discrete_optimization/rcpsp_multiskill/multiskill_to_rcpsp.py

```diff
@@ -42,7 +42,7 @@ def construct_rcpsp_by_worker_type(
         one_worker_type_per_task: bool = False,
     ):
         params_cp = ParametersCP(
-            time_limit=0.5,
+            time_limit=30,
```

Comment:
This value Can be a bit conservative, however it seems the computation time of test unit don't explode so Ok !